model-building.Rmd

```diff
@@ -8,7 +8,7 @@ We will take advantage of the fact that you can think about a model partitioning
 
 For very large and complex datasets this will be a lot of work. There are certainly alternative approaches - a more machine learning approach is simply to focus on the predictive ability of the model. These approaches tend to produce black boxes: the model does a really good job at generating predictions, but you don't know why. This is a totally reasonable approach, but it does make it hard to apply your real world knowledge to the model. That, in turn, makes it difficult to assess whether or not the model will continue to work in the long-term, as fundamentals change. For most real models, I'd expect you to use some combination of this approach and a more classic automated approach.
 
-It's a challenge to know when to stop. You need to figure out when your model is good enough, and when additional investment is unlikely to pay off. I particularly like this quote from reddit user Broseidon241: 
+Es un desafio saber cuando detenerse. Tu necesitas averiguar cuando tu modelo es bastante bueno, y es poco probable que la investigacion adicional pagué. Me gusta especialmente esta cita del usuario Broseidon241: 
```

Comment:
*investment* es inversión. 
La traducción de *pay off* sería valer la pena o dar buenos resultados. 
Por lo tanto, una posible traducción correcta a *and when additional investment is unlikely to pay off* sería *y es poco probable que la inversión valga la pena* o *y es poco probable que la inversión de buenos resultados*

Reply:
Es un desafio saber cuando detenerse. Necesitas darte cuenta cuando tu modelo es lo suficientemente bueno, y cuando no es conveniente invertir mas tiempo en él. Me gusta especialmente esta cita del usuario de reddit Broseidon241:

Reply:
ok.